src/components/Plants/index.tsx

```diff
@@ -68,9 +68,11 @@ export default function PlantsDashboard(props: PlantsDashboardProps): JSX.Elemen
 
   if (!plantingSites) {
     return (
-      <Box sx={{ position: 'fixed', top: '50%', left: '50%' }}>
-        <CircularProgress />
-      </Box>
+      <TfMain>
+        <Box sx={{ position: 'fixed', top: '50%', left: '50%' }}>
```

Comment:
you can remove this and just have `margin='auto'` on the spinner below

Reply:
for future reference what the box positioning does i think:

<img width="367" alt="Untitled presentation - Google Slides 2022-11-29 04-37-41" src="https://user-images.githubusercontent.com/1865174/204531217-6ada46c5-8c9f-42e6-85bf-b152893d6b0e.png">

<img width="340" alt="Untitled presentation - Google Slides 2022-11-29 04-35-34" src="https://user-images.githubusercontent.com/1865174/204530927-7604a927-79cb-4c81-ba06-191cede1e25f.png">